test/Directory.Build.targets

```diff
@@ -14,6 +14,11 @@
     <!-- Put packages for tests in subfolder so we don't try to sign them -->
     <PackageOutputPath>$(PackageOutputPath)tests\</PackageOutputPath>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsTestProject)' == 'true'">
```

Comment:
I added this, otherwise NuGet would warn `.editorconfig` not packed because it starts with `.`
(In dotnet-format test assets we have project file with.editorconfig)